cmd/mayactl/app/command/volume_create.go

```diff
@@ -78,7 +79,10 @@ func (c *CmdVolumeCreateOptions) Validate(cmd *cobra.Command) error {
 // Run does tasks related to mayaserver.
```

Comment:
Can the comments be changed to "Run makes create volume request to maya-apiserver after verifying whether the volume already exists or not. In case if the volume already exists it returns the error and come out of execution."

Reply:
sure will do it .